tmt/steps/discover/fmf.py

```diff
@@ -195,13 +197,13 @@ def wake(self, keys=None):
         super().wake(keys=keys)
 
     @property
-    def is_in_standalone_mode(self):
+    def is_in_standalone_mode(self) -> Any:
```

Comment:
Why is this 'Any' if it has to return bool? Because no all parent classes are typed yet?

Reply:
mypy says it returns `Any` but I agree it should be `bool`, maybe some explicit declaration should be added 
`tmt/steps/discover/fmf.py:204: error: Returning Any from function declared to return "bool"`

Reply:
fixed, thx

Reply:
fixed, please approve